Orange/tests/test__orange.py

```diff
@@ -13,10 +13,6 @@ def test_valuecount(self):
         b = _valuecount.valuecount(a)
         np.testing.assert_almost_equal(b, [[1, 2], [0.6, 0.4]])
 
-        a = np.array([[0, 1, 1, 1], [0.1, 0.2, 0.3, 0.4]])
```

Comment:
Why was this removed? I might be looking at this wrongly, but I don't see where this code is duplicated.